tests/native_tests/contrib/walk.hy

```diff
@@ -58,26 +58,14 @@
   ;; make sure a macro from the current module works
   (assert (= (macroexpand-all '(foo-walk))
              42))
-  (assert (= (macroexpand-all '(with [a 1]))
-             '(with* [a 1])))
+  (assert (= (macroexpand-all '(-> 1 a))
+             '(a 1)))
```

Comment:
These new tests seems unrelated to the current PR

Reply:
The old tests used `with`, which is no longer a macro, so I had to change them to something else.